manifests/06-operator.yaml

```diff
@@ -31,7 +32,7 @@ spec:
           resources:
             requests:
               cpu: 10m
-          imagePullPolicy: Always
```

Comment:
PR https://github.com/openshift/cluster-samples-operator/pull/122 just merged with this change